xbmc/addons/ScreenSaver.cpp

```diff
@@ -13,33 +13,25 @@
 #include "windowing/GraphicContext.h"
 #include "windowing/WinSystem.h"
 
-namespace ADDON
+using namespace ADDON;
+using namespace KODI::ADDONS;
+
+namespace
+{
+static void get_properties(const KODI_HANDLE hdl, struct KODI_ADDON_SCREENSAVER_PROPS* props)
```

Comment:
No need for `static` as this function is in an unnamed namespace, which has exactly the same effect and purpose, to reduce function's visibility to the compile unit.